### PR TITLE
Checkbox: new DS6 styles

### PR DESCRIPTION
**Pull request by PrashantAshok**
## Description
Issue #140 

## Context
This applied DS6 styles to checkboxes, along with making use of new icons. 

screenshot-2018-5-1 ebay skin for ds6 1

### CODE REVIEW

**Comment by seangates** [comment]
Something doesn't look right with the icon. When zoomed out, it looks like it has a heavier top border, and when zooming it in looks like this: 👇 

image

**Comment by ianmcburnie** [comment]
We might need to recreate the SVG from the source files.

**Comment by ianmcburnie** [comment]
Hmm, it looks a lot better now. @PrashantAshok Did you fix this, since we spoke about it yesterday?

screen shot 2018-05-02 at 9 53 50 am

Or perhaps the inline SVG icon is fine, but the background SVG is the broken one.

File: src/less/less/ds6/variables.less [hunk 1]
```diff
@@ -161,3 +161,8 @@
 @ds6-icon-top-seller-height: 20px;
 @ds6-icon-fast-n-free-width: 22px;
 @ds6-icon-fast-n-free-height: 16px;
+
+// Outlines
+//------------------------------------
+
+@outline-dotted: 1px dotted @ds6-color-g205-gray;
```
Comment (ianmcburnie) [inline src/less/less/ds6/variables.less]: We should probably prefix this variable name with `ds6-` to be consistent.

File: src/less/checkbox/ds6/checkbox-mixin.less [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.breadcrumb-mixin() {
```
Comment (seangates) [inline src/less/checkbox/ds6/checkbox-mixin.less]: This says `breadcrumb-mixin`. It should be `checkbox-mixin`.

**Comment by PrashantAshok** [comment]
@seangates @ianmcburnie - The PR is updated with changes from radio.

File: docs/_includes/ds6/checkbox.html [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+                    </svg>
+                </span>
+            </span>
+        </div>
+    </div>
+
+    {% highlight html %}
+<span class="checkbox" style="color: #71c63e">
+    <input class="checkbox__control" type="checkbox" />
+    <span class="checkbox__icon checkbox__icon--inherit" hidden>
```
Comment (ianmcburnie) [inline docs/_includes/ds6/checkbox.html]: We can remove `checkbox__icon--inherit` now.

**Pull request merged**